src/config.js

```diff
@@ -93,9 +94,11 @@ export function buildConfig (options: ConfigOptions = {}): Config {
   const pjson = options.pjson || {}
   const cli: CLI = pjson['cli-engine'] || {}
   const name = options.name || pjson.name || 'cli-engine'
+  const namespaces = options.namespaces || pjson.namespaces
```

Comment:
you don't need to do this since the Object.assign will already default to the options value